docs/Schema-changes.md

```diff
@@ -0,0 +1,164 @@
+### Changes to Aries-vcr issuer schema
+
+The Aries-vcr application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries-vcr application is in the way that credential schemas are issued.  
+  
+Included below are a list of optional changes to credential schemas that issuers may be interested in making. For the purpose of clearly displaying how these changes are being used, we've provided examples of the credential changes in the new Orgbook site.
```

Comment:
"we've provided examples of the credential changes in the new Orgbook site."  Provide a link to the repository so it is clear what code is being used for the examples.